### ISSUE TITLE
VSC WLC friction logging 1.0.14

### ISSUE THREAD

**Issue by ayim** [MicrosoftDocs/intellicode#392]
![image](https://user-images.githubusercontent.com/10951869/178432099-e6813713-e64b-40df-8dd4-27cecac1d935.png)
this becomes
![image](https://user-images.githubusercontent.com/10951869/178432140-38ad0a4a-2fa2-4c14-9983-9355d07127e9.png)

**Comment by ayim** [comment]
File: 

import pandas as pd
import tensorflow as tf
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Conv2D, Flatten, Dropout, MaxPooling2D, GlobalAveragePooling2D
from tensorflow.keras.preprocessing.image import ImageDataGenerator
from tensorflow.python.keras.models import Model
from tensorflow.python.keras.applications import xception

import os
import numpy as np
import matplotlib.pyplot as plt

data = pd.read_csv("data.csv")

batch_size = 64
epochs =

**Comment by ayim** [comment]
Would love to not get 0 here.
Epochs and image height are going to be >0

![image](https://user-images.githubusercontent.com/10951869/178432304-fbd6f429-5384-4e46-b58e-771a396366ae.png)

**Comment by ayim** [comment]
![image](https://user-images.githubusercontent.com/10951869/178432353-21ef1214-8f56-4680-aedd-d861b7ec7418.png)
still 0

**Comment by ayim** [comment]
weird behaviour where I tried to delete the square braces to type 0.1

![image](https://user-images.githubusercontent.com/10951869/178432623-990d7531-2d04-460a-961f-1af16e638b35.png)

**Comment by ayim** [comment]
For this number, it put me into snippet completion, nice.

![image](https://user-images.githubusercontent.com/10951869/178432946-3a3dcb89-4406-4740-9334-ccb64786ed66.png)

**Comment by ayim** [comment]
Repeated suggestion.
Sorry, didn't have logging set to ALL
![image](https://user-images.githubusercontent.com/10951869/178433025-44d08ffb-b824-4040-9482-1d1cf7995eb2.png)

import pandas as pd
import tensorflow as tf
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense, Conv2D, Flatten, Dropout, MaxPooling2D, GlobalAveragePooling2D
from tensorflow.keras.preprocessing.image import ImageDataGenerator
from tensorflow.python.keras.models import Model
from tensorflow.python.keras.applications import xception

import os
import numpy as np
import matplotlib.pyplot as plt

data = pd.read_csv("data.csv")

batch_size = 64
epochs = 20
IMG_HEIGHT = 150
IMG_WIDTH = 150

trainim = ImageDataGenerator(rescale=1./255, horizontal_flip=True, vertical_flip=True, rotation_range = 75, width_shift_range = 0.1, height_shift_range = 0.1, brightness_range = [0.8,1.2], zoom_Range = 0.1)
validation_image_generator = ImageDataGenerator(rescale=1./255)

dataset = pd.read_csv("/content/drive/My Drive/course/train.csv")
dataset

train = dataset.iloc[:4000].reset_index()
train

val = dataset.iloc[4000:].reset_index()

from shutil impmort import impmort

 and

**Comment by ayim** [comment]
Going to close this since we're now on 1.0.15.
